docs/_posts/murat-gunay/2022-02-21-bert_biomed_pubmed_uncased_en.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: model
+title: BERT Biomedical Embeddings
+author: John Snow Labs
+name: bert_biomed_pubmed_uncased
+date: 2022-02-21
+tags: [pubmed, biomedical, bert, microsoft, en, open_source]
+task: Embeddings
+language: en
+edition: Spark NLP 3.4.0
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This embeddings model was imported from `Hugging Face ([link](https://huggingface.co/microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext)). It is pre trained from scratch using abstracts from PubMed and full-text articles from PubMedCentral. This model achieves state-of-the-art performance on many biomedical NLP tasks, and currently holds the top score on the Biomedical Language Understanding and Reasoning Benchmark.
```

Comment:
Unclosed quotes after Hugging Face

Reply:
@josejuanmartinez 

I will fix, but there should be a systematic problem regarding quotes. It was normal on UI

Reply:
@murat-gunay It's in your text, your last commits all have this problem.

Reply:
Actually I put quotes manually each time when I upload model.